data-store/chapter.md

```diff
@@ -88,21 +90,21 @@ Characterizing Failure
 ---------------
 
 Databases are often characterized by how closely they adhere to
-[ACID](http://en.wikipedia.org/wiki/ACID) semantics:
```

Comment:
We don't include links to Wikipedia as footnotes, since it's usually trivially true that you can look something up in Wikipedia, and I think your text has sufficient explanation anyway.